src/JDNCalendarConversion.ts

```diff
@@ -399,8 +399,9 @@ export module JDNConvertibleConversionModule {
 
         const n = truncateDecimals((275 * m)/9) - w * truncateDecimals((m + 9)/12) + d - 30;
         const a = x - 623;
-        const b = truncateDecimals(a/4);
```

Comment:
we implemented `truncateDecimals` according to https://stackoverflow.com/questions/4912788/truncate-not-round-off-decimal-numbers-in-javascript/9232092#9232092, but maybe that is not the same behaviour as in Fortran ...

Reply:
This the current implementation  we are using:

```ts
     * Removes the fraction from a given number (<https://stackoverflow.com/questions/4912788/truncate-not-round-off-decimal-numbers-in-javascript/9232092#9232092>).
     * This also works for negative numbers.
     *
     * 1.2 -> 1
     * -3.2 -> -3
     *
     * @param num the number whose fraction is to be removed.
     * @returns given number without fractions.
     */
    const truncateDecimals =  (num: number): number => {
        return Math[num < 0 ? 'ceil' : 'floor'](num);
    };
```